### PR TITLE
Eshabansiya/realtime session

### PR DESCRIPTION
**Pull request by eshabansiya**
### 🦋 Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

participants/session-start
- If the profile does not have a session set, it will say "Waiting for session to start"
- If it starts with a session, it will display “Session has just started with templateId: {templateId}"
- If a session is updated, it will display “Session has just started with templateId: {templateId}"

- Assumed that sessions will be null until the facilitator assigns each role a session 
- Styling without the button to notify the facilitator 

Realtime: 
- Updated profile table publication settings to allow Realtime API calls 
- Channel to listen to UPDATE in profile

Queries: 
queries/sessions: fetchTemplateId 
- gets template_id from session in session table

### 🦧 Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

Realtime Publication Settings:
Screenshot 2025-11-23 at 8 32 41 PM

Screenshot 2025-11-23 at 3 33 59 PM
Screenshot 2025-11-23 at 3 33 47 PM

## 🐸 How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

## 🐙 Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### 🐁 Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @me-liu

### CODE REVIEW

File: app/participants/session-start/styles.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+export const Heading3 = styled.h3`
+  font-family: ${Sans.style.fontFamily};
+  color: ${COLORS.black70};
+  font-size: 0.7rem;
+  font-style: normal;
+  font-weight: 500;
+`;
+
+export const WelcomeTag = styled.div`
+  dispay: flex;
```
Comment (me-liu) [inline app/participants/session-start/styles.ts]: some unused styles here... also a typo on dispay (i think this is also a typo in some other files in /edit-profile and onboarding)

**Review by me-liu — COMMENTED** [review]
hallo functionality looks perfect

are you able to style like so when the session starts (but only add the 'start game' button, not the 'contact facilitator' button. also no need to add the navbar) 

play game-2
play game

I know most of the exercises don't have names yet but they should in the future, so it's ok if it shows up as blank. [figma](https://www.figma.com/design/ONiX8NmbtginHjmnZjTonb/ECOVET-GLOBAL?node-id=1064-4221&m=dev): go to midfi and the participant flow
